src/Uno.UI/UI/Xaml/Controls/TextBox/TextBoxView.wasm.cs

```diff
@@ -105,6 +106,9 @@ internal void SetTextNative(string text)
 			InvalidateMeasure();
 		}
 
+		internal void Select(int start, int length)
+			=> WebAssemblyRuntime.InvokeJS($"document.getElementById({HtmlId}).setSelectionRange({start}, {start + length})");
```

Comment:
Same comment here about the validation. Also, it'd be best to add a method that does this logic in the `WindowManager.ts` file instead of adding inline javascript.